train:
  ros__parameters:
    environment: 'CarTrack' # CarGoal, CarWall, CarBlock, CarTrack, CarBeat
    track: 'multi_track' # track_1, track_2, track_3, multi_track, multi_track_testing -> only applies for CarTrack
    algorithm: 'TD3'
    max_steps_exploration: 1000
    max_steps_training: 1000000
    reward_range: 2.0
    collision_range: 0.2
    observation_mode: 'lidar_only'
    evaluate_every_n_steps: 2000
    evaluate_for_m_episodes: 3
    # actor_path & critic_path must exist for retrain.sh, it can't be commented
    actor_path: 'rl_logs/multi_track_with_speed12/models/actor_checkpoint.pht'
    critic_path: 'rl_logs/multi_track_with_speed12/models/critic_checkpoint.pht'
#    gamma: 0.95
    # tau: 0.005
    g: 1
    # batch_size: 32
    # buffer_size: 1000000
    actor_lr: 0.0001
    critic_lr: 0.001
    # max_steps: 10
    step_length: 0.1
    # seed: 123
